docs/UserGuide.md

```diff
@@ -142,48 +140,47 @@ RapidTracer offers search functions for both patient and appointment data. Beyon
 <p></p>
 
 <h4 id="find" style="color: #7393B3">
-  Searching for contacts: `find`
+  Searching for contacts: <code>find</code>
 </h4>
 
 Shows a list of contacts in RapidTracer matching the keywords provided. The contact only needs to partially match any of the keywords provided.
 
 Format: `find KEYWORD [MORE_KEYWORDS]`
 
-- The KEYWORD searched is case-insensitive.
-- Order of keywords does not matter.
-- Possible keyword types:
-  - NAME
-  - PHONE_NUMBER
-  - ADDRESS
+- The `KEYWORD` search is case-insensitive.
+- Searches for the following patient information:
+  - `NAME`
+  - `PHONE_NUMBER`
+  - `ADDRESS`
 
 <h4 id="findappt" style="color: #7393B3">
-  Finding appointment: `findappt`
+  Finding appointment: <code>findappt</code>
 </h4>
 
 Shows a list of appointments in RapidTracer matching the keywords provided. The appointment only needs to partially match any of the keywords provided.
 
 Format: `findappt KEYWORD [MORE_KEYWORDS]`
 
-- The KEYWORD searched is case-insensitive.
+- The `KEYWORD` search is case-insensitive.
 - Order of keywords does not matter.
-- Possible keyword types:
-  - NAME
-  - ID
+- Searches for the following appointment information:
+  - `NAME`
+  - `ID` (what's this?)
```

Comment:
I think can delete `ID`, the patient UUID was originally meant to be human readable but this was changed